packages/components/src/organisms/PaymentMethods/PaymentMethods.tsx

```diff
@@ -60,7 +60,7 @@ const PaymentMethods = forwardRef<HTMLDivElement, PaymentMethodsProps>(
               data-fs-payment-methods-flag
               key={`fs-payment-method-${index}-${text}`}
             >
-              <Icon width={32} height={22.5} name={icon} />
```

Comment:
@renatamottam  In this case where `width` and `height` have different values, what size value should we consider?  We're using the first prop value (width).

Reply:
I think we can leave the icon with `20px` and fix the `data-fs-payment-methods-flag` to be a white rectangle. As these icons names didn't change to the new format, we should keep using `width` and `height` here.

Reply:
As we discussed, I'm reverting those changes for now.